src/modules/core/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import ReactSlider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+type Appearance = {
+  theme?: 'primary' | 'danger';
+};
+
+interface Props {
+  value: number;
+  max?: number;
+  limit?: number;
+  appearance?: Appearance;
+  onChange: (val: any) => void;
+}
```

Comment:
This component needs to be a form element, without that it won't work for our use case.

For that we need to have a `name` for it.

Note that this is a required prop.